src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -924,6 +924,39 @@ private function addAssetMapperSection(ArrayNodeDefinition $rootNode, callable $
                             ->info('The directory to store JavaScript vendors.')
                             ->defaultValue('%kernel.project_dir%/assets/vendor')
                         ->end()
+                        ->arrayNode('precompress')
+                            ->info('Precompress assets with Brotli, Zstandard and gzip.')
+                            ->canBeEnabled()
+                            ->children()
+                                ->arrayNode('extensions')
+                                    ->info('Array of extensions to compress.')
+                                    ->prototype('scalar')->end()
+                                    // Loosely based on https://caddyserver.com/docs/caddyfile/directives/encode#match
+                                    ->defaultValue([
+                                        'css',
+                                        'cur',
+                                        'eot',
+                                        'html',
+                                        'js',
+                                        'json',
+                                        'md',
+                                        'otc',
+                                        'otf',
+                                        'proto',
+                                        'rss',
+                                        'rtf',
+                                        'svg',
+                                        'ttc',
+                                        'ttf',
+                                        'txt',
+                                        'wasm',
+                                        'woff',
```

Comment:
woff and woff2 are already compressed file types.

Reply:
Indeed, good catch. Cloudflare and Caddy compress some WOFF MIME types, but they seem to correspond to legacy types from the early days of WOFF that weren't compressed. I'll update the code.

https://developers.cloudflare.com/speed/optimization/content/compression/